main/src/tokenizer/deberta_v2_tokenizer.rs

```diff
@@ -243,9 +243,25 @@ impl Tokenizer<DeBERTaV2Vocab> for DeBERTaV2Tokenizer {
         special_tokens_mask.extend(vec![0; tokens_ids_with_offsets_1.ids.len()]);
         special_tokens_mask.push(1);
         token_segment_ids.extend(vec![0; tokens_ids_with_offsets_1.ids.len() + 2]);
-        output.push(self.vocab.token_to_id(DeBERTaV2Vocab::cls_value()));
+        output.push(
+            self.vocab.token_to_id(
+                self.vocab
+                    .special_token_map
+                    .cls_token
+                    .as_ref()
+                    .expect("CLS token expected for encoding"),
```

Comment:
Yayks! I think that we can avoid this `expect` as I did it in #76:
https://github.com/npatsakula/rust-tokenizers/blob/nllb_support/main/src/vocab/nllb_vocab.rs#L71

Reply:
Thank you - I have reworked the implementation taking inspiration from your proposal - can you please have a look and let me know what you think?